curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo/64fb29348a60361ccd45c1e2.md

```diff
@@ -0,0 +1,404 @@
+---
+id: 64fb29348a60361ccd45c1e2
+title: Step 46
+challengeType: 0
+dashedName: step-46
+---
+
+# --description--
+
+`localStorage` has methods you can use for saving, retrieving, and deleting items. The items you're saving could be any JavaScript data type.
```

Comment:
I would suggest adding a brief definition for local storage here as the first sentence.

Reply:
I defined `localStorage` in step 1.

Do you think it makes sense to define it here again? Or should I remove the definition in step 1 and bring it here?

Reply:
In that case, then I think it is fine to leave this as is 👍